grakn-graph/src/main/java/ai/grakn/graph/internal/AbstractGraknGraph.java

```diff
@@ -136,6 +137,13 @@ public AbstractGraknGraph(G graph, String keyspace, String engine, boolean batch
     }
 
     /**
+     * Opens the thread bound transaction
+     */
+    public void openTransaction(){
+        localIsOpen.set(true);
+    }
```

Comment:
Pretty much yes, but it is not exposed. The purpose is to just maintain a thread bound flag so that we know when to throw an exception is the graph is passed to a different thread.